recipes/pyrfr/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyrfr" %}
+{% set version = "0.8.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 092cb7931486672ad00a71f37cc2e204cdb4b75ab0a47056667cc2683cc07dbd
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - boost
+    - {{ compiler('cxx') }}
+    - doxygen
+    - swig >3.0
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pyrfr
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pyrfr/
+  license: BSD-4-Clause
+  license_file: LICENSE_cereal-1.2.2
```

Comment:
@mfeurer Just to confirm, is this indeed the license of pyrfr? Just because of the file name...

Reply:
That looks to be the license for [`cereal`](https://github.com/USCiLab/cereal/blob/master/LICENSE)

Reply:
That will need to be copied into the recipe folder during the build. If there are more than one I usually copy all 3rd party licenses into a `thirdparty` folder I create in the `$RECIPE_DIR`.

Reply:
The *actual* license might be a bit problematic:
https://github.com/automl/random_forest_run/blob/e51ab097120c25b7a637e48c6cae16351b8db0c5/pyrfr/setup.py#L47

Reply:
Yeah, it‘s just the one linked by GitHub in the sidebar which is why I used it. But true, the one in setup is what we would need.

Reply:
Happy to accept any request you might have but I‘d otherwise wait for @mfeurer to chime in since this might need to be specified upstream and I don‘t really like to guess when it comes to licenses. 😄

Reply:
Do we need to create a new release for the re-licensing: https://github.com/automl/random_forest_run/pull/55 ?

Reply:
I think that would be the best way since conda-forge prefers having the license packed by the upstream maintainers, as required by most licenses (https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually).

Reply:
If it's not possible, let me know as we can also manually inlcude it.

Reply:
I'm currently figuring out how to do that again

Reply:
@franchuterivera This PR is about building a conda package for the pyrfr and requires and update to the pyrfr package on pypi. As building a wheel is already on your stack, do you think it would be much overhead automating the building of a source distribution for pypi? It would be great if that was on github actions, too.

Reply:
Just to add, we can also use a GitHub release for this, as long as it provides a tarball. 
Whether that is hosted in GitHub or PyPI is not too important.

Reference:

- https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos
- https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository

Sample URLs for the simplejson package:
- https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
- https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
- https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz

Reply:
Thanks for pointing this out. That's probably a bit easier to do than uploading to pypi as no credentials are needed. As the upload is the easy part compared to building the package we'll probably go with that.